### DISCUSSION TITLE
Documentation resources

### DISCUSSION THREAD

**Discussion by tonyfast** [jupyter/accessibility#29] [General]
Let us use this thread to accumulate resources about accessibility that help understand the standards, motivations, impact, and technology

**Comment by tonyfast** [comment]
[Why using `tabindex` values greater than “0” is bad](https://karlgroves.com/2018/11/13/why-using-tabindex-values-greater-than-0-is-bad) 

[MDN tabindex docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)

@0618 here are some references i was trying to dig up about tabindex.

**Reply by 0618** [reply]
Thanks, Tony!

**Comment by tonyfast** [comment]
[know you standards - html5accessibility](https://html5accessibility.com/stuff/2020/11/24/know-your-standards/)

[Not so short note on aria-label usage - html5accessibility](https://html5accessibility.com/stuff/2020/11/07/not-so-short-note-on-aria-label-usage-big-table-edition/)
